client/src/api/config/queries.ts

```diff
@@ -20,18 +20,23 @@ const fetchConfig = async () => {
     });
     if (!response.data.connectionInfo) {
       response.data.connectionInfo = JSON.parse(import.meta.env.IQENGINE_CONNECTION_INFO ?? null);
+      console.log("Setting connectionInfo env: ", response.data.connectionInfo)
```

Comment:
Should we be logging connection info or google analytics key?

Reply:
Strong point, I'll change it to only log if it's null